portal/config/eproms/QuestionnaireBank.json

```diff
@@ -180,8 +180,8 @@
     },
     {
       "classification": "recurring",
-      "due": "{\"days\": 14}",
-      "expired": "{\"days\": 29}",
+      "due": "{\"days\": 0}",
+      "expired": "{\"months\": 3, \"days\": -15}",
```

Comment:
@pbugni  I thought you were preferring additive date math - why not make this consistent w/ line 91 ( "expired": "{\"months\": 2, \"days\": 13}",)

Reply:
Otherwise, I think this looks good...

Reply:
I was playing with different values - additive and subtraction, trying to find the best combo.  The one checked in is pretty good, but it breaks the logic in `visit_name`, as we only look at the month.  (i.e. 3 month looks like 2 and 6 looks like 5 with this).